web/src/app/core/services/repository.service.ts

```diff
@@ -30,7 +31,8 @@ export class RepositoryService {
     private afs: AngularFirestore,
     private fns: AngularFireFunctions,
     private authService: AuthenticationService,
-    private activityService: ActivityService
+    private activityService: ActivityService,
+    private httpClient: HttpClient
```

Comment:
I think `HttpClient` is no longer needed?

Reply:
correct. I will remove it